src/client/pfs_file.go

```diff
@@ -1,76 +1,70 @@
 package client
 
 import (
+	"archive/tar"
 	"context"
 	"io"
+	"io/ioutil"
 	"time"
 
 	"github.com/pachyderm/pachyderm/v2/src/internal/errors"
+	"github.com/pachyderm/pachyderm/v2/src/internal/errutil"
 	"github.com/pachyderm/pachyderm/v2/src/internal/grpcutil"
 	"github.com/pachyderm/pachyderm/v2/src/internal/storage/renew"
+	"github.com/pachyderm/pachyderm/v2/src/internal/tarutil"
 	"github.com/pachyderm/pachyderm/v2/src/pfs"
 )
 
-// AppendFile appends a file.
-func (c APIClient) AppendFile(repo, commit, path string, overwrite bool, r io.Reader, tag ...string) error {
-	mfc, err := c.NewModifyFileClient(repo, commit)
-	if err != nil {
-		return err
-	}
-	if err := mfc.AppendFile(path, overwrite, r, tag...); err != nil {
-		return err
-	}
-	return mfc.Close()
+// PutFile puts a file into PFS from a reader.
+func (c APIClient) PutFile(repo, commit, path string, r io.Reader, opts ...PutFileOption) error {
+	return c.WithModifyFileClient(repo, commit, func(mfc ModifyFileClient) error {
+		return mfc.PutFile(path, r, opts...)
+	})
 }
 
-// AppendFileTar appends a set of files from a tar stream.
-func (c APIClient) AppendFileTar(repo, commit string, overwrite bool, r io.Reader, tag ...string) error {
-	mfc, err := c.NewModifyFileClient(repo, commit)
-	if err != nil {
-		return err
-	}
-	if err := mfc.AppendFileTar(overwrite, r, tag...); err != nil {
-		return err
-	}
-	return mfc.Close()
+// PutFileWriter sets up a writer for putting a file in PFS.
+func (c APIClient) PutFileWriter(repo, commit, path string, opts ...PutFileOption) (io.WriteCloser, error) {
+	return nil, errV1NotImplemented
```

Comment:
What's the deal here?  If this is not in v2, just remove the method.

Reply:
I wanted to keep it around because it seems like a useful method that we would want to implement eventually, but it probably makes more sense to remove it for now. I'll go ahead and remove it and add a TODO to the 2.0 GitHub project.